ur_robot_driver/launch/ur10_bringup.launch

```diff
@@ -13,20 +13,7 @@
   <arg name="limited" default="false" doc="Use the description in limited mode (Every axis rotates from -PI to PI)"/>
   <arg name="headless_mode" default="false" doc="Automatically send URScript to robot to execute. On e-Series this does require the robot to be in 'remote-control' mode. With this, the URCap is not needed on the robot."/>
 
-  <include file="$(find ur_robot_driver)/launch/ur_common.launch">
-    <arg name="debug" value="$(arg debug)"/>
+  <include file="$(find ur_robot_driver)/launch/ur_common.launch" pass_all_args="true">
     <arg name="use_tool_communication" value="false"/>
```

Comment:
It could be nice to move `use_tool_communication` to the list of `args` at the top of the file and instead of providing a `default`, just set the `value` (like it's done here).

As it is right now though it does clearly communicate the fact that for these robots (ie: non-e-Series), `use_tool_communication` cannot be set to anything but `false`. Perhaps adding a comment explaining *why* this is would be good, to avoid users getting confused when they try to edit the `.launch` file and force it to `true`.

Reply:
Should we incorporate this here or a separate PR?

Reply:
I would do it in a separate one (as it's a different subject).

But you're the maintainer, so you could *request changes*.

Reply:
Let's keep this separate, then.